packages/cli/src/commands/help.ts

```diff
@@ -0,0 +1,212 @@
+import chalk from 'chalk';
+import stripAnsi from 'strip-ansi';
+import { LOGO, NAME } from '@vercel-internals/constants';
+
+const INDENT = ' '.repeat(2);
+const NEWLINE = '\n';
+const MAX_LINE_LENGTH = process.stdout.columns;
+
+export interface CommandOption {
+  name: string;
+  shorthand: string | null;
+  type: 'boolean' | 'string';
+  argument?: string;
+  deprecated: boolean;
+  description: string;
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** allow intentionally undocumented‏

```suggestion
  description?: string;
```